contracts/credits/src/bin/schema.rs

```diff
@@ -0,0 +1,11 @@
+use cosmwasm_schema::write_api;
+
+use credits::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+
+fn main() {
+    write_api! {
```

Comment:
so schema is there, it's just in a different format, because of newer cosmwasm schema lib used (Used 1.2.1 instead of 1.0.0)
Changes were made in this release:
https://github.com/CosmWasm/cosmwasm/blob/main/CHANGELOG.md#120---2023-01-24

As for schema generated files, they are in gitignore from project generator. Do we need them?

Reply:
across all the repos we have these schema .json files. and I'm sure that at least Ilya uses them in order to get familiar with contracts' interface. so they belong in the repo for sure